FlingEngine/Gameplay/src/FirstPersonCamera.cpp

```diff
@@ -0,0 +1,83 @@
+#include "FirstPersonCamera.h"
+#include "FlingMath.h"
+#include "pch.h"
+
+namespace Fling 
+{
+    FirstPersonCamera::FirstPersonCamera(float aspectRatio) :
+		m_front(glm::vec3(-1.0f))
+    {
+		m_nearPlane   =   0.1f;
+		m_farPlane    =   1000.0f;
+		m_speed       =   100.0f;
```

Comment:
The speed might be a good thing to have in the `EngineConf.ini` file in the future. Feels a little fast right now